src/utils/bivariate/bivariateLegendUtils.ts

```diff
@@ -74,13 +49,9 @@ export function createBivariateMeta(
   yDenominator: string,
   stats: Stat,
 ): LayerMeta {
-  const xAxisNominatorIndicator = stats.indicators.find(
-    (ind) => ind.name === xNumerator,
-  );
+  const xAxisNominatorIndicator = stats.indicators.find((ind) => ind.name === xNumerator);
```

Comment:
Nominator -> Numerator? maybe add a linter rule that "Nominator" is not expected anywhere in codebase?

Reply:
ah, that's the place where I've got this! 
yeah, lemme search it through the codebase

Reply:
<img width="179" alt="Screenshot 2022-12-02 at 16 33 43" src="https://user-images.githubusercontent.com/20676525/205304549-325f1096-2102-4cdb-ae9c-415c57c56807.png">